### PR TITLE
add addSbtPlugin to build sbt assembly

### PR DESCRIPTION
**Pull request by junqueira**
References #xxxx

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @junqueira,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by junqueira** [comment]
> Hi @junqueira,
> 
> Thank you for your contribution! We really value the time you've taken to put this together.
> 
> Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:
> 
> http://www.lightbend.com/contribute/cla
ok

**Comment by junqueira** [comment]
Thank, sign the CLA.

File: project/assembly.sbt [hunk 1]
```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6")
```
Comment (junqueira) [inline project/assembly.sbt:1]: sbt-assembly

Comment (renatocaval) [reply project/assembly.sbt:1]: Hi @junqueira, can you explain why we should have sbt-assembly added to this project?

Comment (junqueira) [reply project/assembly.sbt:1]: I believe it is the default wheel "sbt assembly" and everything works as expected, or am I wrong?

**Comment by junqueira** [comment]
I believe it is the default wheel "sbt assembly" and everything works as expected, or am I wrong?

**Comment by ihostage** [comment]
@junqueira Why `akka-projection` needs fat Jar? It's just a library.

**Comment by renatocaval** [comment]
@junqueira, sbt assembly only makes sense if you are building an application for which you have a Main class. You package the whole application, all jars, in one single fat jar. Then you only need to execute the jar. 

This is totally not the case for a library like akka-projection. Here we build a jar that you can include in your application. If you want to, you can package your application with sbt assembly. 

sbt assembly is not a "default wheel", on the contrary. Most projects won't need it. 

I'm closing this PR since this is clearly a misunderstanding of what sbt-assembly is used for.

**Pull request closed without merging**